lib/tasks/export.rake

```diff
@@ -274,14 +274,18 @@ def export_paper(paper)
       end
     end
     paper.tasks.each do |task|
+      next unless task.answers.any? ||
+          task.comments.any? ||
+          task.try(:invitations).try(:any?)
+
       # Skip any tasks that have been snapshotted, they should be in
       # the version directories.
       next if Snapshot.find_by(source: task).present?
-      next if task.answers.size == 0 && task.comments.size == 0
+
       zip_add_rendered_html(zos,
                             "#{prefix}/#{task.title.parameterize}-task.html",
                             nil,
-                            'export/generic_answers.html.erb',
```

Comment:
it seems like there is a lot of overlap between `export/generic_answers.html.erb` and `export/task.html.erb` that could be dried up, but this is just a one-off export task so I'm not that worried.